toolkit/tools/imagecustomizerapi/config.go

```diff
@@ -41,7 +41,18 @@ func (c *Config) IsValid() (err error) {
 		if err != nil {
 			return fmt.Errorf("invalid 'os' field:\n%w", err)
 		}
-		hasResetBootLoader = c.OS.ResetBootLoaderType != ResetBootLoaderTypeDefault
+		hasResetBootLoader = c.OS.BootLoader.Reset != ResetBootLoaderTypeDefault
```

Comment:
I think the meaning of `ResetBootLoaderTypeDefault` is `ResetBootLoaderTypeUndefined` - I wonder if we should rename that constant... (I know it's not part of this change - but maybe it improves clarity)